apk-tools.yaml

```diff
@@ -50,12 +57,14 @@ subpackages:
     dependencies:
       runtime:
         - apk-tools
+
   - name: "lua5.3-apk"
     description: "Lua module for libapk"
     pipeline:
       - runs: |
           mkdir -p "${{targets.subpkgdir}}"/usr
           mv "${{targets.destdir}}"/usr/lib "${{targets.subpkgdir}}"/usr/lib
+
 update:
   enabled: true
```

Comment:
Depending on how long we want to use this version, we should (IIUC) consider setting this to `false` now to avoid auto updates

Reply:
2.14.0 final is due out in a couple of weeks.  I don't think that we should disable auto-updates, right?

Reply:
I just mean that since the `Wolfictl Check Updates` check failed, I think it will automatically open a PR to "bump" this to `2.12.11` as soon as this PR gets merged, which might be confusing to folks and even accidentally get merged

Reply:
Got it.  Now done :)